parlai/agents/hugging_face/dict.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+from parlai.core.dict import DictionaryAgent
+
+try:
+    from transformers import GPT2Tokenizer
+except ImportError as e:
+    raise e(
+        'Need to install Hugging Face transformers repository. '
+        'Try `pip install transformers`.'
+    )
+from abc import ABC, abstractmethod
+
+SPECIAL_TOKENS = {
+    "bos_token": "<bos>",
+    "eos_token": "<eos>",
+    "pad_token": "<pad>",
+}
+
+
+class HuggingFaceDictionaryAgent(DictionaryAgent, ABC):
```

Comment:
There's a general TODO for us to be aware of later in the half. DictionaryAgent should become a truly abstract, and then we should take the current DA and call it a ParlaiDictionaryAgent and have all our years of grossness into it.

Definitely not for now